### DISCUSSION TITLE
C++ STL总结 | 行码棋

### DISCUSSION THREAD

**Discussion by anda522** [anda522/anda522.github.io#73] [General]
https://wyqz.top/p/870124582.html 

 C++ STL 超全总结-基于算法竞赛（悠享版）本文介绍常用STL知识，注重应用，强调用法，不强调原理和繁杂的记忆。看过之后请多运用，多敲代码试。

费尽心思重新梳理了一下，注意了些美观性，修改了部分错误，添加了部分解释，编写过程非常难。

**Comment by anda522** [comment]
## 总结了很多STL知识，有什么错误可以提出来，我尽量改正。如果有缺漏的，可以在下面评论区留言，我看到了会尽快回复。

**Comment by anda522** [comment]
> @HuiyangChan
> __builtin_popcount 函數下面的代碼框裏寫了 __builtin_ffs(x)

抱歉，小错误，感谢指出，稍后会进行修改。

**Comment by stong12345** [comment]
作者 你好，关注你写的文章几年了，看到你的STL2025年更新了，在哪里能够下载最新的PDF哇，文章后面的下载地址下载的PDF感觉不是最新的呢

**Reply by anda522** [reply]
最新的文档可能还没有整理出来呢，最近太忙了，我公众号里面放的文档就是最新的，之后我尽量找时间再整理一下吧。但是公众号上面放的其实和最新的也没有什么大的改动，不影响看的。
